roles/db_server/templates/onconfig.csmsaga

```diff
@@ -123,8 +123,8 @@ LTXEHWM 80
 # CONSOLE      - The path of the Informix console message file
 ###################################################################
 
-MSGPATH $INFORMIXDIR/tmp/online.log
-CONSOLE $INFORMIXDIR/tmp/online.con
+MSGPATH {{ informix_log_file }}
```

Comment:
if we do not wish to create separate path for informix log, the we can use default /tmp/online.log